common/src/configuration/global.rs

```diff
@@ -532,6 +533,15 @@ fn convert_node_config(
         .map(|seconds| seconds as u64)
         .map_err(|e| ConfigurationError::new(&key, &e.to_string()))?;
 
+    // The cooldown period between balance enquiry checks in seconds; requests faster than this will be ignored.
+    // For specialized wallets processing many batch transactions this setting could be increased to 60 s to retain
+    // responsiveness of the wallet with slightly delayed balance updates (default: 1)
+    let key = "wallet.balance_enquiry_cooldown_period";
```

Comment:
nobody is ever going to change this to anything other than 1. I would say hardcoding it as 1 second is better in this case

Reply:
The default is set to 1. The configuration setting is needed for stress testing or for running a large wallet where many events occur that can change the balance, where 1 s is not a good choice. These are average time measurements for the call to the `pub async fn get_balance` SQL queries to complete:
- ~15,000 total spent and unspent TXOs: 1 s
- ~125,000 total spent and unspent TXOs: 400 s